bindings/c/c_ffi.cpp

```diff
@@ -129,6 +129,15 @@ char *pcsx2ipc_version(PCSX2Ipc *v, bool batch) {
     }
 }
 
+PCSX2Ipc::EmuStatus pcsx2ipc_status(PCSX2Ipc* v, bool batch) {
+    if (batch) {
+        v->Status<true>();
```

Comment:
return v->Status<true> and not 0

Reply:
That won't compile, `v->Status<true>()` returns a char* while `v->Status<false>();` returns an EmuStatus.